all_test.go

```diff
@@ -17,8 +17,19 @@ import (
 //go:embed test/data/case18/assets
 var assets embed.FS
 
+func setupFileCopyMethod(m *testing.M) {
+	// Allow running all the tests with a different FileCopyMethod.
+	// We want to be able to have full coverage no matter the method.
```

Comment:
Regarding you're using `t.Skip`, it is not the "coverage" you mean. Cases which should return error should return error. Users can avoid the case, but we should not skip the test. Please do not skip.

Reply:
I'll change the `t.Skip` to check for an error instead, if that is preferred?

Reply:
If it should return an error, let's do so.